app/ERC821Contracts/index.js

```diff
@@ -0,0 +1,4 @@
+export {
```

Comment:
Like @mertcelebi had done before, can you make the folder structure `contracts/erc821/index.js`? Just leaving the option for adding `721` or others later.